prombench/Makefile

```diff
@@ -37,50 +30,96 @@ cluster_delete:
 		-v CLUSTER_NAME:${CLUSTER_NAME} -v PR_NUMBER:${PR_NUMBER} \
 		-f manifests/cluster_${PROVIDER}.yaml
 
+# /prombench <...> --bench.directory
+BENCHMARK_DIRECTORY := $(if $(BENCHMARK_DIRECTORY),$(BENCHMARK_DIRECTORY),manifests/prombench)
+# /prombench <...> --bench.version
+BENCHMARK_VERSION := $(if $(BENCHMARK_VERSION),$(BENCHMARK_VERSION),master)
+PROMBENCH_GIT_REPOSITORY ?= git@github.com:prometheus/test-infra.git
+PROMBENCH_DIR ?= .
+
+# maybe_pull_custom_version allows custom benchmarking as designed in
+# https://github.com/prometheus/proposals/pull/41. It allows calling
+# /prombench <release> --bench.version=<@commit or branch> which will cause
+# prombench GH job on Prometheus repo to call infra CLI with the non-master BENCHMARK_VERSION.
+# In such a case we pull a prombench repository for the given branch or commit version
+# and adjust PROMBENCH_DIR. As a result `make deploy` and `make clean` jobs
+# will apply /manifests/ apply custom manifests or even node pools.
+.PHONE: maybe_pull_custom_version
```

Comment:
LOL, I just noticed ☎️ XD

It should be [`.PHONY:`](https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html) but I copied other commands 🙈 will fix later.